api/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "pretty": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitReturns": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
```

Comment:
I'll have to read a bit what these rules do.

All in all, just a note that almost all changes to configs that we do on the `webiny-js` repo level, should also be propagated to user's project (`cwp-template-aws` package). Not 100% of the time, but mostly that will be the case.

Maybe you already did it but I just didn't get to that part of the PR. Still, I decided to mention it so I don't forget.